bot.go

```diff
@@ -1,36 +1,379 @@
 package main
 
 import (
+	//"bytes"
+	"encoding/json"
+	"errors"
+	"io/ioutil"
 	"log"
+	"os"
+	"strings"
 	"time"
 
 	tba "gopkg.in/tucnak/telebot.v2" //telegram bot api
 )
 
-type bot struct {
-	api *tba.Bot
+//Bot is main structure with access to api, users data and necessary channels
+type Bot struct {
+	api           *tba.Bot
+	flashcards    map[int64]map[string]map[string]string
```

Comment:
Dalej mi to brzydko wygląda. Możesz zdefiniować swój typ na chat ID, dzięki temu:
* nie pojebie się w miejscu, w którym powinnien być `int`/nie użyjesz przez przypadek `int` w miejscu, gdzie ma być chat ID
* będzie bardziej przejrzyste, co tu trzymasz i jaka jest struktura tych danych.

No i to wciąż będzie mapa. Zastanów się na ile to czytelne, ale mógłbyś zrobić np. tak:

```golang
type chatid int64
type topic string
type flashcards map[string]string

flashcards map[chatid]map[topic]flashcards
```

Moim zdaniem to trochę lepiej się czyta.